.github/PULL_REQUEST_TEMPLATE_nolabels

```diff
@@ -0,0 +1,13 @@
+## ATTENTION - Release Note ack required!
+
+Update the below named `release-note-needed` block to one of:
+
+**\`\`\`release-note**
+**\`\`\`release-note-action-required**
```

Comment:
I'd like to add release-note-experimental also, for notes going into the Experimental Features category.